.github/workflows/ci.yml

```diff
@@ -43,13 +43,13 @@ jobs:
 
       - name: Pytest - PyVortex
         run: |
-          rye run maturin develop --skip-install
-          rye run pytest --benchmark-disable test/
+          uv run maturin develop --skip-install
```

Comment:
I don't think we need Maturin develop?`uv run` should do it automatically no?

Reply:
yeah definetly looks like it, removing